### ISSUE TITLE
Django 2.1 compatibility.

### ISSUE THREAD

**Issue by katsika** [shtalinberg/django-el-pagination#124]
Seems like `django-el-pagination==3.2.4` is not compatible with django-2.1.

I get:

```
Environment:


Request Method: GET
Request URL: http://127.0.0.1:8000/logbook/

Django Version: 2.1
Python Version: 3.7.0


Template error:
In template /Users/ME/Sites/my_site/ascents/templates/logbook/listing/wrapper_paginate.html, error at line 7
   'int' object has no attribute 'encode'
   1 : {% load el_pagination_tags %}
   2 : {% lazy_paginate object_list %}
   3 :
   4 : {% for ascent in object_list %}
   5 :   {% include 'logbook/listing/ascent.html' %}
   6 : {% endfor %}
   7 :  {% show_more %}
   8 :

Traceback:

File "/Users/ME/Sites/my_site/my_siteenv/lib/python3.7/site-packages/django/core/handlers/exception.py" in inner
  34.             response = get_response(request)

File "/Users/ME/Sites/my_site/my_siteenv/lib/python3.7/site-packages/django/core/handlers/base.py" in _get_response
  156.                 response = self.process_exception_by_middleware(e, request)

File "/Users/ME/Sites/my_site/my_siteenv/lib/python3.7/site-packages/django/core/handlers/base.py" in _get_response
  154.                 response = response.render()

File "/Users/ME/Sites/my_site/my_siteenv/lib/python3.7/site-packages/django/template/response.py" in render
  106.             self.content = self.rendered_content

File "/Users/ME/Sites/my_site/my_siteenv/lib/python3.7/site-packages/django/template/response.py" in rendered_content
  83.         content = template.render(context, self._request)

File "/Users/ME/Sites/my_site/my_siteenv/lib/python3.7/site-packages/django/template/backends/django.py" in render
  61.             return self.template.render(context)

File "/Users/ME/Sites/my_site/my_siteenv/lib/python3.7/site-packages/django/template/base.py" in render
  171.                     return self._render(context)

File "/Users/ME/Sites/my_site/my_siteenv/lib/python3.7/site-packages/django/test/utils.py" in instrumented_test_render
  96.     return self.nodelist.render(context)

File "/Users/ME/Sites/my_site/my_siteenv/lib/python3.7/site-packages/django/template/base.py" in render
  937.                 bit = node.render_annotated(context)

File "/Users/ME/Sites/my_site/my_siteenv/lib/python3.7/site-packages/django/template/base.py" in render_annotated
  904.             return self.render(context)

File "/Users/ME/Sites/my_site/my_siteenv/lib/python3.7/site-packages/django/template/loader_tags.py" in render
  150.             return compiled_parent._render(context)

File "/Users/ME/Sites/my_site/my_siteenv/lib/python3.7/site-packages/django/test/utils.py" in instrumented_test_render
  96.     return self.nodelist.render(context)

File "/Users/ME/Sites/my_site/my_siteenv/lib/python3.7/site-packages/django/template/base.py" in render
  937.                 bit = node.render_annotated(context)

File "/Users/ME/Sites/my_site/my_siteenv/lib/python3.7/site-packages/django/template/base.py" in render_annotated
  904.             return self.render(context)

File "/Users/ME/Sites/my_site/my_siteenv/lib/python3.7/site-packages/django/template/loader_tags.py" in render
  62.                 result = block.nodelist.render(context)

File "/Users/ME/Sites/my_site/my_siteenv/lib/python3.7/site-packages/django/template/base.py" in render
  937.                 bit = node.render_annotated(context)

File "/Users/ME/Sites/my_site/my_siteenv/lib/python3.7/site-packages/django/template/base.py" in render_annotated
  904.             return self.render(context)

File "/Users/ME/Sites/my_site/my_siteenv/lib/python3.7/site-packages/django/template/loader_tags.py" in render
  188.             return template.render(context)

File "/Users/ME/Sites/my_site/my_siteenv/lib/python3.7/site-packages/django/template/base.py" in render
  173.                 return self._render(context)

File "/Users/ME/Sites/my_site/my_siteenv/lib/python3.7/site-packages/django/test/utils.py" in instrumented_test_render
  96.     return self.nodelist.render(context)

File "/Users/ME/Sites/my_site/my_siteenv/lib/python3.7/site-packages/django/template/base.py" in render
  937.                 bit = node.render_annotated(context)

File "/Users/ME/Sites/my_site/my_siteenv/lib/python3.7/site-packages/django/template/base.py" in render_annotated
  904.             return self.render(context)

File "/Users/ME/Sites/my_site/my_siteenv/lib/python3.7/site-packages/django/template/library.py" in render
  214.         _dict = self.func(*resolved_args, **resolved_kwargs)

File "/Users/ME/Sites/my_site/my_siteenv/lib/python3.7/site-packages/el_pagination/templatetags/el_pagination_tags.py" in show_more
  363.             default_number=data['default_number'])

File "/Users/ME/Sites/my_site/my_siteenv/lib/python3.7/site-packages/el_pagination/utils.py" in get_querystring_for_page
  174.         return '?' + querydict.urlencode()

File "/Users/ME/Sites/my_site/my_siteenv/lib/python3.7/site-packages/django/http/request.py" in urlencode
  524.                 for v in list_

File "/Users/ME/Sites/my_site/my_siteenv/lib/python3.7/site-packages/django/http/request.py" in <genexpr>
  524.                 for v in list_

Exception Type: AttributeError at /logbook/
Exception Value: 'int' object has no attribute 'encode'
```

**Comment by adrysn** [comment]
Probably fixed in Django 2.1.1?
https://code.djangoproject.com/ticket/29627

**Comment by katsika** [comment]
Fixed indeed, thanks. Django's regression.
